packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
+import VirtualizedList from '../../VirtualizedList';
+import CellTitle from '../../VirtualizedList/CellTitle';
+import CellActions from '../../VirtualizedList/CellActions';
+
+function ListToVirtualizedList(props) {
+	const { id, items, columns, titleProps } = props;
+	if (!titleProps.actionsKey) {
+		titleProps.actionsKey = 'actions';
+	}
+	// Backward compatibility: find array in object attr:
+	const supposedActions = {};
+	if (props.items.length > 0) {
+		const item = props.items[0];
+		Object.keys(item).forEach((key) => {
```

Comment:
Reduce in this case is not really improving the readability. But we still can improve that : 
```
Object.keys(item)
	.filter(key => Array.isArray(item[key]))
	.forEach((key) => { supposedActions[key] = true; });
``